### PR TITLE
completed react-hash-routing

### PR DESCRIPTION
**Pull request by m-Cho1**
react-hash-routing

### CODE REVIEW

File: react-hash-routing/client/components/header.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default function Header(props) {
+  return (
+    <header className="mb-5">
+      <nav className="navbar navbar-dark bg-dark shadow-sm">
+        <div className="container">
+          <div className="col px-0">
+            {/* this anchor should go back to the catalog at '#' */}
+            <a href="" className="navbar-brand">
```
Comment (scbowler) [inline react-hash-routing/client/components/header.jsx:10]: Although it works as an empty string it is best to put a # as the href for the "Home" route. It shows intention and that you didn't accidentally leave it empty.
```JSX
<a href="#" className="navbar-brand">
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just one small update needed. Let me know if you have any questions.

**Comment by m-Cho1** [comment]
I completely forgot to check on that section, my bad! Edited header.jsx

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**